components/Carbon.js

```diff
@@ -83,6 +85,29 @@ class Carbon extends PureComponent {
         'Shift-Tab': 'indentLess'
       }
     }
+
+    let background = '';
+    if (!config.didThemeChange) {
```

Comment:
Used to keep track of whether a theme was changed...without this check, the background will not change between different themes